dashboard/constants.js

```diff
@@ -1,37 +1,41 @@
-module.exports =  {
-    serverAddr: {
-        ip : "0.0.0.0",
-        port : 33333
-    },
-    hvBone: {
-       ip: "192.168.1.146",
-       port: 9091,
-    },
-    lvBone: {
-       ip: "192.168.1.140",
-       port: 9091
-    },
-    databaseAddr: {
-       ip: "127.0.0.1",
-       port: 27017
-    },
-    scanningRate : 300,
-    mins:{
-       off:{},
-       idle:{},
-       readyForPumpdown:{},
-       pumpdown:{},
-       ready:{},
-       postRun:{},
-       crawl:{}
-    },
-    maxs:{
-        off:{},
-        idle:{},
-        readyForPumpdown:{},
-        pumpdown:{},
-        ready:{},
-        postRun:{},
-        crawl:{}
-     }  
-}
\ No newline at end of file
+/*
+Author: Eric Udlis
+Purpose: Store all constants in one place
+*/
+module.exports = {
+  serverAddr: {
+    ip: "127.0.0.1",
```

Comment:
Why'd we change this back to 127.0.0.1? Do your internal tests still work if we have it listening on all interfaces?

Reply:
Oops, didn't notice that, 127.0.0.1 or 0.0.0.0 works for my internal tests so I guess I just didn't notice